libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -120,6 +126,34 @@ export interface IExtractorProjectConfiguration {
   dependenciesToExclude?: string[];
 }
 
+/**
+ * The extractor configuration for individual dependencies.
+ *
+ * @public
+ */
+export interface IExtractorDependencyConfiguration {
+  /**
+   * The name of dependency
+   */
+  dependencyName: string;
+  /**
+   * The version of dependency
+   */
+  dependencyVersion: string;
```

Comment:
We should make this a semver version range

Reply:
Nice idea, have changed to use semver version range.

Reply:
Let's also rename it to `dependencyVersionRange` as well to make it clear that it accepts a semver range.